config/config.proto

```diff
@@ -90,6 +90,16 @@ message FlowdockConfig {
   optional bool send_resolved = 4 [default = false];
 }
 
+// Configuration for notification via OpsGenie.
+message OpsGenieConfig {
+  // https://api.opsgenie.com/v1/json/alert
+  optional string webhook_url = 1;
```

Comment:
This URL should probably not be configurable here, but as a global flag (similar to how we do it with most of the others in the file, except for Slack - Slack seems to need auth/identification params in the URL, but those could actually be appended dynamically too and the prefix statically configured).